cli/tests/integrations/test_package.py

```diff
@@ -671,6 +669,46 @@ def test_search_table():
     assert stderr == b''
 
 
+def test_search_ends_with_wildcard():
```

Comment:
@jsancio: can you arbitrate this so we don't run into this issue in future PRs?

The question is whether we should allow minor fixes that are otherwise irrelevant to the PR.  Including:

a) docstring fixes
b) pydoc fixes
c) code movement (like this one)
d) naming inconsistencies
e) typos 

@ssk2 mentioned that he's fine with this, but wary of 40 lines changes like this, but this is really just one "thing", since it's a copy+paste.  I'm of course against an irrelevant real 40-line change as well.

As discussed, the tradeoff is:
+ less friction to fixing problems
- more difficult `git revert`

Since I run into these sorts of issues very regularly, and since I can only think of two times we've run `git revert` in the four months I've been here, I think it's clearly beneficial to allow this.  Otherwise, I'm disincentivized from fixing these issues.  I'll fix a minor issue if I can do it immediately, but not if I have to 1) commit my changes to the current branch, 2) create a new branch, 3) run syntax checks, 4) push, 5) write a new PR, 6) switch back to the original branch and try to remember where I was.